documentation/development-docs/generate-format.ps1xml-file.md

```diff
@@ -180,18 +180,18 @@ namespace Microsoft.Azure.Commands.Profile.Models
 
 // code omitted for brevity
 
-        [Ps1Xml(Label = "Subscription Id", Target = ViewControl.Table)]
+        [Ps1Xml(Label = "SubscriptionId", Target = ViewControl.Table)]
```

Comment:
Why are the spaces being removed from the column headings?

Reply:
It's recommended to give a real property name so customer may use it for a next query, for example
```
PS C:\> Get-AzSubscription

SubscriptionId                       SubscriptionName          State   TenantId
---------------                      -----------------         -----   ---------
c9cbd920-c00c-427c-852b-c329e824c3a8 Azure SDK Powershell Test Enabled 72f988bf-86f1-41af-91ab-7a64d1d63df5
6b085460-5f21-477e-ba44-4cd9fbd030ef Azure SDK Infrastructure  Enabled 72f988bf-86f1-41af-91ab-7a64d1d63df5

PS C:\> Get-AzSubscription | ? {$_.SubscriptionName  -like "*Test*" }
```